prog/main.go

```diff
@@ -158,12 +218,14 @@ func logCensoredArgs() {
 
 func main() {
 	var (
-		flags         = flags{}
-		mode          string
-		debug         bool
-		weaveEnabled  bool
-		weaveHostname string
-		dryRun        bool
+		flags                       = flags{}
+		mode                        string
+		debug                       bool
+		weaveEnabled                bool
+		weaveHostname               string
+		dryRun                      bool
+		containerLabelFlagsPositive containerLabelFiltersPositive
+		containerLabelFlagsNegative containerLabelFiltersNegative
```

Comment:
Instead of positive and negative please call them `containerLabelFlags` and `containerLabelFlagsExclude`